content/applications/inventory_and_mrp/inventory/management/products/strategies.rst

```diff
@@ -1,75 +1,135 @@
+
+In Odoo, there are two strategies for automatically replenishing inventory: :guilabel:`Reordering
+Rules` and the :guilabel:`Make to Order (MTO)` route. Although these strategies differ slightly,
+they both have similar consequences: triggering the automatic creation of a purchase or
+manufacturing order. The choice of which strategy to use depends on the business's manufacturing and
+delivery processes.
 
-Minimum Stock rules and Make to Order have similar consequences but
-different rules. They should be used depending on your manufacturing and
-delivery strategies.
 
 Terminology
 ===========
 
-Minimum stock rule
-------------------
+Replenishment report and reordering rules
+-----------------------------------------
+
+The :guilabel:`Replenishment Report` is a list of all products that have a negative forecast
+quantity.
+
+:guilabel:`Reordering Rules` are used to ensure that there's always a minimum amount of a product in
+stock in order to manufacture products and/or fulfill sales orders. When the stock level of a
+product reaches its minimum, Odoo automatically generates a purchase order with the quantity needed
+to reach the maximum stock level.
 
-Minimum Stock rules are used to ensure that you always have the minimum
-amount of a product in stock in order to manufacture your products
-and/or answer to your customer needs. When the stock level of a product
-reaches its minimum the system will automatically generate a procurement
-order with the quantity needed to reach the maximum stock level.
+Reordering rules can be created and managed in the replenishment report or from the product form.
 
-Make to Order
+Make to order
 -------------
 
-The Make to Order function will trigger a Purchase Order of the amount
-of the Sales Order related to the product. The system will **not** check
-the current stock. This means that a draft purchase order will be
-generated regardless of the quantity on hand of the product.
+:guilabel:`Make to Order (MTO)` is a procurement route that creates a draft purchase order or
+manufacturing order each time a sales order is confirmed, *regardless of the current stock level*.
+Unlike products replenished using reordering rules, Odoo automatically links the sales order to the
+purchase order (PO) or manufacturing order (MO) generated by the MTO route. Another difference
+between reordering rules and MTO is with MTO, Odoo generates a draft PO or MO immediately after the
+sales order is confirmed. With reordering rules, Odoo generates a draft PO or MO when the product's
+forecasted stock falls below the set minimum quantity. In addition, Odoo will automatically add
+quantities to the PO/MO as the forecast changes, as long as the PO/MO is not confirmed.
+The MTO route is the best replenishment strategy for products that are customized and/or no stock is
+kept on hand.
+
 
 Configuration
 =============
 
-Minimum stock rules
--------------------
+Replenishment report and reordering rules
```

Comment:
This was tough, because the first one was about the terminology and the second is configuration- so I was hoping people would see the higher level category and realize that particular "Replenishment report and reordering rules" section was only for the Configuration or Terminology sections.